src/NHibernate/Proxy/AbstractProxyFactory.cs

```diff
@@ -10,8 +11,9 @@ namespace NHibernate.Proxy
 	/// <summary>
 	/// Convenient common implementation for ProxyFactory
 	/// </summary>
-	public abstract class AbstractProxyFactory: IProxyFactory
+	public abstract class AbstractProxyFactory: IProxyFactory, ISerializable
```

Comment:
The easiest (and I think the _correct_) way to serialize a proxy was to serialize its proxy factory.